packages/playwright-core/src/server/page.ts

```diff
@@ -445,7 +445,7 @@ export class Page extends SdkObject {
     const contextOptions = this._browserContext._options;
     return {
       media: this._emulatedMedia.media || 'no-override',
-      colorScheme: this._emulatedMedia.colorScheme !== undefined ? this._emulatedMedia.colorScheme : contextOptions.colorScheme ?? 'light',
+      colorScheme: this._emulatedMedia.colorScheme !== undefined ? this._emulatedMedia.colorScheme : contextOptions.colorScheme ?? 'no-preference',
```

Comment:
Looks like it was changed to 'light' in https://github.com/microsoft/playwright/commit/ba29e99acec0462809a605ffb3d9e507b0fac083#diff-e2488cd41c99b3beafe59ebbf87b9ac81dcbdd1e6f6e363cfd016905eea3e1cf. @mxschmitt do you remember why?

I believe this entire return statement should be rewritten to something like:
```js
    return {
      media: this._emulatedMedia.media ?? 'no-override',
      colorScheme: this._emulatedMedia.colorScheme ?? contextOptions.colorScheme ?? 'no-override',
      reducedMotion: this._emulatedMedia.reducedMotion ?? contextOptions.reducedMotion ?? 'no-override',
      forcedColors: this._emulatedMedia.forcedColors ?? contextOptions.forcedColors ?? 'no-override',
    };
```